### DISCUSSION TITLE
How to work with 'RichText/Media_Album'?

### DISCUSSION THREAD

**Discussion by mihalt** [Terrance/SkPy#234] [Q&A]
Hi! I found type of messages 'RichText/Media_Album' witch should contain few images at once. Response contains only "albumId" from useful information. So, as I understand, I should use separate api to get this images by this "albumId". Do you know what is api?

**Comment by Terrance** [comment]
As noted before, you'll have to see what Skype for Web does with those kinds of messages.

**Comment by mihalt** [comment]
I tried, but don’t see anything in network requests that contains this api
key, when I click on show another images.
